make-rules/shared-macros.mk

```diff
@@ -162,6 +162,17 @@ endif
 PYTHON_VERSION =	2.7
 PYTHON_VERSIONS =	2.7
 
+PYTHON2_VERSIONS = 2.7
+PYTHON2_VERSION = 2.7
+PYTHON2_RUNTIME_PKG = runtime/python-$(subst .,,$(PYTHON2_VERSION))
+
+PYTHON3_VERSIONS = 3.4 3.5 3.7
+PYTHON3_VERSION	= 3.5
```

Comment:
Do you want to remove 3.4 later? Otherwise I would only expect PYTHON3_VERSIONS = 3.5 3.7

Reply:
Yes, I will remove Python 3.4 itself in a second PR.